<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choosestack</class>
 <widget class="QWidget" name="choosestack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flashcards</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Flashcards</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="stacksListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonFirstRowLayout">
     <item>
      <widget class="QPushButton" name="newPushButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openPushButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsSecondRowLayout">
     <item>
      <widget class="QPushButton" name="learnPushButton">
       <property name="text">
        <string>Learn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editPushButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPushButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="settingsPushButton">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="aboutPushButton">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
